pkg/apis/core/types_cloudprofile.go

```diff
@@ -98,6 +98,11 @@ type MachineImage struct {
 	Name string
 	// Versions contains versions, expiration dates and container runtimes of the machine image
 	Versions []MachineImageVersion
+	// UpdateStrategy is the update strategy to use {patch, minor, major} for the machine image
+	//  - patch: update to the latest patch version of the current minor version.
+	//  - minor: update to the latest minor and patch version.
+	//  - major: always update to the overall latest version (default).
```

Comment:
```suggestion
	// UpdateStrategy is the update strategy to use for the machine image. Possible values are:
	//  - patch: update to the latest patch version of the current minor version.
	//  - minor: update to the latest minor and patch version.
	//  - major: always update to the overall latest version (default).
```

The same suggestion applies to the doc string in the versioned API (v1beta1).